Prototypes/BeleensMerge.cs

```diff
@@ -0,0 +1,497 @@
+
+                case "Baby Red Dragon":
+                    if (!Core.IsMember)
+                    {
+                        Core.Logger($"{req.Name} requires Membership to Obtain");
+                        return;
+                    }
+                    Adv.BuyItem("AriaPet", 12, req.Name);
+                    break;
+
+                case "Zealith Reavers":
+                    Core.FarmingLogger($"{req.Name}", quant);
+                    if (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        LegionExercise3.Exercise(new[] { "Judgement Hammer", "Legion Token" });
+                        LegionExercise4.Exercise(new[] { "Judgement Scythe", "Legion Token" });
+                        CoreLegion.FarmLegionToken(50);
+                        Adv.BuyItem("Underworld", 238, req.Name);
+                    }
+                    break;
+
+                case "Great Astral Wings":
+                    Core.FarmingLogger($"{req.Name}", quant);
+                    Core.EquipClass(ClassType.Farm);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("Elemental", "Mana Falcon", req.Name, isTemp: false);
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    break;
+
+                case "Wave Cutter":
+                    Core.FarmingLogger($"{req.Name}", quant);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("Pirates", "Shark Bait", req.Name, isTemp: false);
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    break;
+
+                case "Star Caster Staff":
+                    Adv.BuyItem("Castle", 48, req.Name);
+                    break;
+
+                case "Scarlet's Costume":
+                    if (!Core.IsMember)
+                    {
+                        Core.Logger($"{req.Name} requires Membership to Obtain");
+                        return;
+                    }
+                    Adv.BuyItem("Sleuthhound", 65, req.Name);
+                    break;
+
+                case "Infernal Dark Blade of Cruelty":
+                    if (!Core.IsMember)
+                    {
+                        Core.Logger($"{req.Name} requires Membership to Obtain");
+                        return;
+                    }
+                    Adv.BuyItem("Battleon", 10, req.Name);
+                    break;
+                case "Chaos Dragonlord Helm":
+                    Core.FarmingLogger($"{req.Name}", quant);
+                    Core.EquipClass(ClassType.Farm);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("Stalagbite", "Vath", req.Name, isTemp: false);
```

Comment:
vath can be a bit wonky there is a bit in Corebots > line 1256: "Core.KillVath":
`public void KillVath(string? item = null, int quant = 1, bool isTemp = false, bool log = true, bool publicRoom = false)`
please use that instead 👍